### ISSUE TITLE
Installing A32NX updates fails if the remote MCDU server is running

### ISSUE THREAD

**Issue by rseg** [flybywiresim/installer#340]
### Installer Version

v2.2.2

### Describe the bug

If the remote MCDU server is still running, the FlyByWire Installer shows a "Download failed!" message when finishing the installation of an updated version of A32NX.

### Expected behavior

The FlyByWire Installer should warn that the remote MCDU server is running before updating, just like it's telling that the simulator is running.

Alternatively, the installer can offer to stop the remote MCDU server before updating the A32NX version.

### Steps to reproduce

1. Start server.exe in the "./flybywire-aircraft-a320-neo/MCDU SERVER" folder.
2. Start the FlyByWire Installer
3. Update the selected version or switch versions
4. Wait for the update to finish
5. The following message will appear:

![download_failed](https://user-images.githubusercontent.com/8848654/147873119-1863101d-998e-4616-a4d1-923e3e4633d6.png)

**Comment by woodss** [comment]
Just to add - I just came here to report this because it was failing silently while trying to install the development version and I had no idea why it wasn't completing. 

I only noticed the issue occurring because I tried starting from scratch and when removing (manually) the a320 folder from Community, it told me "files were in use" - of course they were, the MCDU server was still running. 

Pretty obvious really but definitely needs a notification in the app (similar to the one that warns you to close MSFS).
